tests/cli/test_rasa_data.py

```diff
@@ -152,6 +153,87 @@ async def mock_from_importer(importer: TrainingDataImporter) -> Validator:
         data.validate_files(args)
 
 
+def test_rasa_data_augment_nlu(run: Callable[..., RunResult],):
+
+    out_path = "data/augmentation_results"
+    os.makedirs(out_path)
+
+    data_root = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
```

Comment:
We use the pathlib fixture to get access to temporary paths. Also simplifies the directory operations. See [here](https://github.com/RasaHQ/rasa/blob/e1a71efb8add2577dcf8149328588fcbeb462c41/tests/nlu/featurizers/test_count_vectors_featurizer.py) for reference.